DatabaseFunctionsGenerator/CodeHelpers/RetrofitInstance.java

```diff
@@ -0,0 +1,20 @@
+﻿package !--packageName--!;
+
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class RetrofitInstance {
+    public  static Retrofit GetRetrofitInstance()
+    {
+        Retrofit retrofit;
+        Retrofit.Builder builder;
+
+        builder = new Retrofit.Builder()
+                .baseUrl("!--url--!");

```

Comment:
Can you do this in cs `` ;.someFunction();``?

Reply:
Yes, it can be done